Doc/library/sqlite3.rst

```diff
@@ -1362,40 +1334,10 @@ The following example illustrates the implicit and explicit approaches:
 .. literalinclude:: ../includes/sqlite3/converter_point.py
 
 
-Default adapters and converters
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-There are default adapters for the date and datetime types in the datetime
-module. They will be sent as ISO dates/ISO timestamps to SQLite.
-
-The default converters are registered under the name "date" for
-:class:`datetime.date` and under the name "timestamp" for
-:class:`datetime.datetime`.
-
-This way, you can use date/timestamps from Python without any additional
-fiddling in most cases. The format of the adapters is also compatible with the
-experimental SQLite date/time functions.
-
-The following example demonstrates this.
-
-.. literalinclude:: ../includes/sqlite3/pysqlite_datetime.py
-
-If a timestamp stored in SQLite has a fractional part longer than 6
-numbers, its value will be truncated to microsecond precision by the
-timestamp converter.
-
-.. note::
-
-   The default "timestamp" converter ignores UTC offsets in the database and
-   always returns a naive :class:`datetime.datetime` object. To preserve UTC
-   offsets in timestamps, either leave converters disabled, or register an
-   offset-aware converter with :func:`register_converter`.
-
-
 .. _sqlite3-adapter-converter-recipes:
 
 Adapter and Converter Recipes
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
IMO, particularly when framed as Guides/How-tos, it might make more sense to split the adapter and converter recipes/defaults and move them under their respective task-oriented "Using adapters" and "Converting SQLite values" to connect them to the high-level task and make each top-level heading correspond to the task they describe.

Reply:
I'm not sure about this. This implies we need to split the "default adapters/converters" section as well. They are both very small, so I'm not sure of there is added value for the user.

Reply:
I'll keep it like it is for this PR. We can always split it up later. Marking this as resolved for this PR.